saveModul/saveFile.js

```diff
@@ -1,15 +1,16 @@
 const mongoose = require('mongoose');
 const fs = require('fs');
 
+const Issue = require('../models/issueModel');
 const FacadeFileController = require('../facade/fileFacadeFunctions.js');
 
-exports.saveFile = (file, issueId) => {
+exports.saveFile = (file) => {
```

Comment:
One task of this module is to save the file to the storage (remote storage, disk, whatever..). You shouldn't be accessing model directly from here. Only dals is accessing model. Let's chat live.